source/reference/beta-program.txt

```diff
@@ -12,13 +12,70 @@
    :depth: 2
    :class: singlecol
 
-Content for this page will be created in DOCSP-38285.
+.. include:: /includes/beta-program-intro.rst
+
+Each ``mongosync`` release will have a corresponding {+c2c-beta-program-short+} 
```

Comment:
[suggestion]

Consider using the present tense here. Spitballing: "Each ``mongosync`` release has a corresponding...".